### DISCUSSION TITLE
Trouble with integrated circuit compiling

### DISCUSSION THREAD

**Discussion by Helena-NWE** [MrTJP/ProjectRed#1839] [Q&A] [answered]
I don't know where I should be asking about this so I am doing it here.
I understand this circuit is returning errors when compiling cuz the multiple gates aren't going through a or gate, but there is only a single input to the pink output. why is it returning a error?
![image](https://github.com/MrTJP/ProjectRed/assets/76846084/4b78dbab-c8c1-49f0-a49d-a10aa1fe0d2f)

**Comment by MrTJP** [comment, marked as answer]
Good question! Because of the red alloy wire being used to connect several colors to pink, you are essentially driving pink from any gate that drives those colors. Circuit designs that work in-world won't necessarily work in the editor due to compilation rules like this. 

Better error overlays to help debug issues like this coming soon.
